### ISSUE TITLE
Errors when submitting feedback

### ISSUE THREAD

**Issue by parlarjb** [rackerlabs/encore-ui#753]
We're having users report that they're getting "An error occurred submitting your feedback" when using the built-in feedback submission.

One of the reports we've heard:

> "Trying to submit feedback from Account Overview page in Encore, window pops up fine – enter the information.  Next it tries to email you the feedback via my local mail client which I don’t use (tries to open mac mail).  I know previous feedback I have submitted it auto sent from the back end. Here is the screen shot of error."

**Comment by CITguy** [comment]
#### TL;DR
The directive attempts to submit feedback via a non-existent API endpoint and resorts to sending an email if that fails. Since the hardcoded endpoint doesn't exist, it always fails the API call (displaying an error in the UI), and brings up the local mail client for feedback submission.


#### Details
The `rxFeedback` directive first attempts to send feedback via the defined api endpoint `/api/feedback`, and resorts to sending via email if that fails.  After consulting with @halkon, we found that the `/api/feedback` endpoint does not seem to exist in the nginx proxy configurations in any environment... ever.

Would that endpoint exist and be functional, the displayed error message would contain additional details of the failure.  However, because of the missing endpoint, it defaults to `"An error occurred submitting your feedback"` (with no additional information), and resorts to the email method.

Encore Ticket Queue currently has an endpoint that is compatible with this directive, and to the best of my knowledge, is the only team using the API functionality.

#### Suggestions
* Short-term fix: the directive needs modified to ignore the API call if not explicitly defined by the app.
* Long-term fix: discuss setting up a centralized feedback API to handle the interaction and provide a better user experience.
* Documentation for the directive should also be updated to reflect its functionality.

**Comment by glynnis** [comment]
@kobowley says this is being handled in EOD.
